src/alfasim_score/converter/alfacase/_tests/test_output_results.py

```diff
@@ -1,27 +1,22 @@
-import json
-import pandas as pd
-import pytest
-from barril.units import Scalar
 from pathlib import Path
 from pytest_regressions.file_regression import FileRegressionFixture
 
 from alfasim_score.common import AnnulusLabel
-from alfasim_score.converter.alfacase.score_output_generator import ScoreOutputGenerator
+from alfasim_score.converter.alfacase.alfasim_score_converter import AlfasimScoreConverter
 
 
 def test_generate_output_file_results(
     shared_datadir: Path, datadir: Path, file_regression: FileRegressionFixture
 ) -> None:
     alfasim_results_directory = shared_datadir / "case.data"
-    well_start_position = Scalar(2072, "m")
-    active_annuli = [AnnulusLabel.A, AnnulusLabel.B, AnnulusLabel.C]
-    # It was defined to use 6 wall layers as output from ALFAsim
-    layers = list(range(6))
-    output_generator = ScoreOutputGenerator(
-        alfasim_results_directory, well_start_position, active_annuli, layers
-    )
-    output_generator.element_name = "7-SRR-2-RJS (2022-07-28_15-01-27)"
+    # dummy input file just to have the reader for this test
+    score_input_filepath = shared_datadir / "score_input_natural_flow.json"
     output_filepath = datadir / "output_score.json"
-    output_generator.generate_output_file(datadir / "output_score.json")
-    output_content = output_filepath.read_text(encoding="utf-8")
+    converter = AlfasimScoreConverter(score_input_filepath, output_filepath)
```

Comment:
I didn't get why this class was created. The previous why to convert the output was good enough.
The only point that could be modified was to remove the requirement of `active_annuli` and `layers`.
Morever, know it is passing the input file, thus why you still need to pass `element_name` and `active_annuli`?

Reply:
> why you still need to pass element_name and active_annuli

We don't, I'm just setting them in the test because the result file used in this test was generated running a case from `plugin-apb`.

Reply:
Nice!